noir-projects/noir-contracts/contracts/docs/docs_example_contract/src/types/card_note.nr

```diff
@@ -17,7 +18,17 @@ pub struct CardNote {
 // docs:end:state_vars-CardNote
 
 impl CardNote {
-    pub fn new(points: u8, randomness: Field, owner: AztecAddress) -> Self {
+    pub fn new(points: u8, owner: AztecAddress) -> Self {
```

Comment:
Refactored CardNote here such that we don't need to pass in randomness which allows me to then not expose randomness on the struct.